drivers/windows/R/dide_auth.R

```diff
@@ -23,6 +23,13 @@ windows_authenticate <- function(call = NULL) {
 
   username <- check_username(
     readline_with_default("DIDE username", windows_guess_username()))
+  if (grepl(" ", username)) {
+    cli::cli_abort(
+      c("Usernames must not contain spaces.",
```

Comment:
Change this perhaps to reflect the user error and not the Auth error. So "The username you have provided does not look valid as it contains {invalid}" with some logic to report back the specific complaint

Reply:
Try now?